src/main/java/org/apache/commons/lang3/annotation/ToEqualsExclude.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.commons.lang3.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Use this annotation to builds a equals excluding the annotated field.
+ */
+@Retention(RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ToEqualsExclude {
```

Comment:
@britter sorry but I can't understand I'll try explaining for you the use of this annotation.
Suppose you have a class and you need use EqualsBuilder or ToStringBuilder or HashCodeBuilder, but you have a field and this field not is necessary in equals and/or tostring and/or hashcode you can note this with @ToStringExclude  and/or @ToEqualsExlucde and/or @ToHashCodeExclude. If one of this annotations is a inner class of EqualsBuilder, ToStringBuilder or HashCodeBuilder when use the apache commons need note using @EqualsBuilder.ToEqualsExclude, @ToStringBuilder.ToStringExclude and  @HashCodeBuilder.ToHashCodeExclude and not only note @ToEqualsExclude, @ToStringExclude and  @ToHashCodeExclude.
Can you understand?